kernel/kselftest.py

```diff
@@ -61,7 +61,7 @@ def setUp(self):
         detected_distro = distro.detect()
         deps = ['gcc', 'make', 'automake', 'autoconf', 'rsync']
         if (self.comp == "powerpc"):
-            if (detected_distro.arch != 'ppc64' or detected_distro.arch != 'ppc64le'):
+            if (detected_distro.arch != 'ppc64' and detected_distro.arch != 'ppc64le'):
```

Comment:
at given time, the ppc OS can be BE mode i.e ppc64 or LE mode i.e ppc64le   so  it will contradict with in power

Reply:
I would suggest to use 
`if 'ppc' not in detected_distro.arch:`